src/lpython/parser/parser.yy

```diff
@@ -243,33 +244,21 @@ units
     ;
 
 script_unit
-    : statement sep   { $$ = SCRIPT_UNIT_STMT($1); }
-    | expr sep        { $$ = SCRIPT_UNIT_EXPR($1); }
-    ;
-/*
-statements
-    : single_stmt_list TK_NEWLINE { }
-    | sep TK_INDENT statements1 TK_DEDENT { }
+    : statement END_OF_FILE
     ;
 
-statements1
-    : statements1 statement sep { }
-    | statement sep { }
+statements
+    : sep TK_INDENT statement sep TK_DEDENT { $$ = $3; }
```

Comment:
This line parses only one statement either a single-line statement (or maybe multi-line)
For example:
```python
if True:
    pass
elif True:
    pass
elif True:
    pass
elif True:
    pass
else:
    pass
```

Reply:
The AST generated for the above example:
```console
$ lpython --show-ast --new-parser --tree examples/expr2.py
└-Module
  |-body=↧
  | └-If
  |   |-test=ConstantBool
  |   | |-value=.true.
  |   | └-kind=()
  |   |-body=↧
  |   | └-Pass
  |   └-orelse=↧
  |     └-If
  |       |-test=ConstantBool
  |       | |-value=.true.
  |       | └-kind=()
  |       |-body=↧
  |       | └-Pass
  |       └-orelse=↧
  |         └-If
  |           |-test=ConstantBool
  |           | |-value=.true.
  |           | └-kind=()
  |           |-body=↧
  |           | └-Pass
  |           └-orelse=↧
  |             └-If
  |               |-test=ConstantBool
  |               | |-value=.true.
  |               | └-kind=()
  |               |-body=↧
  |               | └-Pass
  |               └-orelse=↧
  |                 └-Pass
  └-type_ignores=↧
```
which is identical with the old-parser (CPython)